src/cli/onefuzz/api.py

```diff
@@ -274,6 +274,61 @@ def list(self) -> List[responses.ContainerInfoBase]:
         self.logger.debug("list containers")
         return self._req_model_list("GET", responses.ContainerInfoBase)
 
+    def reset(
```

Comment:
This should use:

`container_type: Optional[List[ContainerType]] = None`

Reply:
Can you add a helper method to ContainerType that indicates the types that are reset by default?

Reply:
updated and used same default as defined in earlier in `reset --container`